### PR TITLE
abi_fix to be verified

### PR DESCRIPTION
**Pull request by pbg-intel**
to verify ddi table abi compatibility issue

### CODE REVIEW

**Comment by bmyates** [comment]
there is already a fix for this issue posted here https://github.com/oneapi-src/level-zero-spec/pull/312

File: scripts/templates/helper.py [hunk 1]
```diff
@@ -895,8 +895,8 @@ def get_class_function_objs_exp(specs, cname):
                     exp_objects.append(obj)
                 else:
                     objects.append(obj)
-    objects = sorted(objects, key=lambda obj: (float(obj.get('version',"1.0"))*10000) + int(obj.get('ordinal',"100")))
-    exp_objects = sorted(exp_objects, key=lambda obj: (float(obj.get('version',"1.0"))*10000) + int(obj.get('ordinal',"100")))              
+    objects = sorted(objects, key=lambda obj: (float(obj.get('version',"1.0").translate({ord('.'):None}))*10000) + int(obj.get('ordinal',"100")))
+    exp_objects = sorted(exp_objects, key=lambda obj: (float(obj.get('version',"1.0").translate({ord('.'):None}))*10000) + int(obj.get('ordinal',"100")))
```
Comment (bmyates) [inline scripts/templates/helper.py:899]: this would translate 2.0 -> 20 which would be less than 1.10 -> 110 but greater than 1.1 -> 11

Comment (rwmcguir) [reply scripts/templates/helper.py:899]: If at all possible, I highly recommend avoiding anything to do with the Ordinal, it is a mess waiting to happen...

**Comment by rwmcguir** [comment]
Closed due to fix by another commit: 
Commit e8a03a2

**Pull request closed without merging**